yt/admin/dashboards/yt_dashboards/common/postprocessors.py

```diff
@@ -94,6 +94,12 @@ def postprocess(self, tags, sensor_name=None):
             tags[GrafanaSystemTags.Rate] = True
             sensor_name = sensor_name[:-5]
         sensor_name = sensor_name.replace(".", "_")
+
+        if SystemFields.LegendFormat in tags:
+            value = tags[SystemFields.LegendFormat]
+            value = value.replace(ContainerTemplate, "{{pod}}")
```

Comment:
`ContainerTemplate -> {{pod}}` is very YT specific replacement, makes sense to have it in yt postprocessor